templates/profile.php

```diff
@@ -114,6 +114,28 @@ function pmpromd_wp_title($title, $sep)
 			return $title;
 		}
 		add_filter("wp_title", "pmpromd_wp_title", 10, 2);
+
+		/**
+		 * We're working with the menu now so remove the filters
+		 */
+		function pmpromd_remove_filters_menu_title( $nav_menu, $args ) {	
+		    remove_filter("wp_title", "pmpromd_wp_title", 10, 2);
+		    remove_filter("the_title", "pmpromd_the_title", 10, 2);
+		    return $nav_menu;
+		}
+		add_filter( 'pre_wp_nav_menu', 'pmpromd_remove_filters_menu_title', 10, 2 );
+
+		/**
+		 * We're done working with the menu so add those filters back
+		 */
+		function pmpromd_readd_filters_menu_title( $items, $args ) {
+		    // we are done working with menu, so add the title filter back
+		    add_filter("wp_title", "pmpromd_wp_title", 10, 2);
+		    add_filter("the_title", "pmpromd_the_title", 10, 2);
+		    return $items;
+		}
+		add_filter( 'wp_nav_menu_items', 'pmpromd_readd_filters_menu_title', 10, 2 );

```

Comment:
```suggestion
		add_filter( 'wp_nav_menu_items', 'pmpromd_readd_filters_menu_title' );

```

Reply:
Just curious why we're removing the priority and argument values here 👀

Reply:
Because the second argument is never used so we don't need to request it and it keeps things simple in implementation.